Topic7/src/main/java/com/aguirre/topic7/controllers/StudentController.java

```diff
@@ -0,0 +1,46 @@
+package com.aguirre.topic7.controllers;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import com.aguirre.topic7.models.Course;
+import com.aguirre.topic7.models.Student;
+import com.aguirre.topic7.models.StudentCourseDao;
+import com.aguirre.topic7.models.StudentDao;
+import com.google.gson.Gson;
+
+@RestController
+@RequestMapping("/students")
+public class StudentController {
+
+	@Autowired
+	private StudentDao studentDao;
+
+	private StudentCourseDao studentCourseDao;

```

Comment:
You should set an @autowired annotation in studentCourseDao too to inject the StudentCourseDao